docs/sources/installation/rhel.md

```diff
@@ -119,6 +119,20 @@ Docker runtime files, or make other customizations, read our Systemd article to
 learn how to [customize your Systemd Docker daemon options](/articles/systemd/).
 
 
+## Red Hat Enterprise Linux 7 Uninstall Docker
```

Comment:
Minor thing (very) but as you have @thaJeztah's comments already, `Uninstall` should be lowercase.   Why? Our style guidance is sentence case --- proper nouns still capped of course.